tools/render-test/shader-input-layout.cpp

```diff
@@ -673,10 +683,43 @@ namespace renderer_test
                 maybeParseOptions(parser, val.Ptr());
                 return val;
             }
-            // TODO: We can include combined texture/sampler cases
-            // here, but we don't really have tests for them and
-            // it might be better to save it for a point where
-            // we support hierarchical binding values more directly.
+            else if (parser.AdvanceIf("TextureSamplerSampler1D"))
```

Comment:
SamplerSampler?

Reply:
Oops. Find-and-replace bit me there. Good catch.